### PR TITLE
iftop: new package at 1.0pre4

### PR DESCRIPTION
**Pull request by wael444**
## Description of package

Top-like tool to display bandwidth usage on an interface

## Installed manifest of package

```
/var/db/kiss/installed/iftop/version
/var/db/kiss/installed/iftop/sources
/var/db/kiss/installed/iftop/manifest
/var/db/kiss/installed/iftop/depends
/var/db/kiss/installed/iftop/checksums
/var/db/kiss/installed/iftop/build
/var/db/kiss/installed/iftop/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man8/iftop.8
/usr/share/man/man8/
/usr/share/man/
/usr/share/
/usr/bin/iftop
/usr/bin/
/usr/

```

## New package

- [X] Latest upstream version.
- [X] I agree to maintain this package.
- [X] I agree to notify the @kiss-community maintainers if I can no longer maintain this package.

## Existing package

- [X] I am the maintainer of this package.

### CODE REVIEW

File: community/iftop/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+export DESTDIR="$1"
```
Comment (ioraff) [inline community/iftop/build:3]: this isn't necessary

**Comment by ioraff** [comment]
why not the stable release?

**Comment by wael444** [comment]
> why not the stable release?

https://www.ex-parrot.com/pdw/iftop/download/?M=D

last release was 1.0pre4, in 2015, stable release is 0.17 yes but it was in 2005.

there is https://code.blinkace.com/pdw/iftop. but i wasn't able to use it because of the git server being down.

File: community/iftop/build [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+export CFLAGS="$CFLAGS -fcommon"
+
+./configure \
+	--prefix=/usr \
+	--sbindir=/usr/bin \
+	--mandir=/usr/share/man
```
Comment (ioraff) [inline community/iftop/build:8]: https://kisscommunity.bvnf.space/kiss/style-guide/#0202

**Comment by wael444** [comment]
differences from 1.0pre4:

+ Missing tui.h header          
+ Cleanup libpcap on exit       
+ fixes for compilation warnings
+ Fix handling of packets that are too short to be valid
+ Support for unlimited number of lines of output in text mode
+ Fix bug with DNS resolution                                 
+ Choose first running interface, rather than first "up" interface
+ Support scales beyond 1Gbps       

i will have to stick with 1.0pre4 purely because the git server is very unstable.

File: community/iftop/build (hunk 2 above)
Comment (wael444) [reply community/iftop/build:8]: owww, i was using spaces in my code editor but packaging for void-packages made me switch fully to using tabs

**Pull request merged**